lib/inferno_template/metadata.rb

```diff
@@ -5,14 +5,89 @@ class Metadata < Inferno::TestKit
     id :inferno_template
     title 'Inferno Template'
```

Comment:
I think we should make all fields feel as real as possible.  E.g. Inferno Example Test Kit.  or 'Inferno Template Test Kit'.